docker/candlepin-base-cs8/setup-devel-env.sh

```diff
@@ -81,6 +81,9 @@ bundle install --without=proton --retry=5 --verbose --full-index
 # Installs all Java deps into the image, big time saver
 ./gradlew --no-daemon dependencies
 
+# Fix issue with owner on mount volume
```

Comment:
Does this issue happen only on cs8 and cs9? Or does it also need to be fixed in c7 (because older branches still need that)?